tests/benchmarks/benchmark_cpu_small.py

```diff
@@ -455,26 +436,19 @@ def run(eq):
 @pytest.mark.benchmark
 def test_LinearConstraintProjection_build(benchmark):
     """Benchmark LinearConstraintProjection build."""
+    eq = desc.examples.get("W7-X")
 
-    def setup():
+    def run():
```

Comment:
moving everything to the run function is changing what its actually profiling, this now includes a bunch of other stuff besides building the linear constraints, is that what we want?

Reply:
The extra stuff is just building individual constraints and objectives, right? Previously, this was effectively just benchmarking `factorize_linear_constraints`. Do we usually pass built constraints to the `LinearConstraintProjection`? Then I can revert it.